api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -168,6 +168,10 @@ message ExternalProcessor {
   // :ref:`override_message_timeout <envoy_v3_api_field_service.ext_proc.v3.ProcessingResponse.override_message_timeout>`
   // If not specified, by default it is 0, which will effectively disable the ``override_message_timeout`` API.
   google.protobuf.Duration max_message_timeout = 10;
+
+  // If false, allows the processor response to allow route cache clearing.
+  // If true, route cache clearing is ignored. False, by default, to ensure backwards compatibility.
```

Comment:
I think the description can be clarified a bit:
```
Prevents clearing the route-cache when the clear_route_cache<link> field is set in an external processor response. If true, the response field is ignored. By default, this field is set to false, to ensure backwards compatibility.
```

Reply:
Done.